src/InboundServer/handlers.js

```diff
@@ -455,52 +534,131 @@ const getCallbackById = async(ctx) => {
     }
 };
 
+const metrics = async (ctx) => {
+    ctx.response.status = 200;
+    ctx.response.body = Metrics.getMetricsForPrometheus();
+};
 
 const map = {
     '/': {
-        get: healthCheck
+        get: {
+            handler: healthCheck,
+            id: 'inbound_health_check',
+            enableSpan: false
+        }
     },
     '/participants/{ID}': {
-        put: putParticipantsById
+        put: {
+            handler: putParticipantsById,
+            id: 'inbound_put_participants_id',
+            enableSpan: true
+        }
     },
     '/participants/{Type}/{ID}': {
-        put: putParticipantsByTypeAndId,
-        get: getParticipantsByTypeAndId
+        put: {
+            handler: putParticipantsByTypeAndId,
+            id: 'inbound_put_participants_type_id',
+            enableSpan: true
```

Comment:
`enalbeSpan` -- should probably be --> `enableTracing` <--- tracing is the important part, we will create spans to enable the tracing.

Let me know what you think.

Same comment applies to the rest of the map entries.

Reply:
updated